src/test/scala/mesosphere/marathon/api/validation/RunSpecValidatorTest.scala

```diff
@@ -92,34 +92,35 @@ class RunSpecValidatorTest extends UnitTest with ValidationTestLike {
       testValidId("/trailing/")
     }
 
-    "single dots in id '/test/.' pass schema and validation" in {
-      testInvalid("/test/.")
-    }
-
-    "single dots in id '/./not.point.less' pass schema and validation" in {
-      testInvalid("/./not.point.less")
-    }
-
-    // non-absolute paths (could be allowed in some contexts)
-    "relative id 'relative/asd' passes schema but not validation" in {
-      val app = AppDefinition(
-        id = PathId("relative/asd"),
-        role = "*",
-        cmd = Some("true"))
-
-      validAppDefinition(app) should haveViolations("/id" -> "Path needs to be absolute")
-
-    }
-
-    // non-absolute paths (could be allowed in some contexts)
-    "relative id '../relative' passes schema but not validation" in {
-      val app = AppDefinition(
-        id = PathId("../relative"),
-        role = "*",
-        cmd = Some("true"))
-
-      validAppDefinition(app) should haveViolations("/id" -> "Path needs to be absolute")
-    }
+    // TODO AN: What to do about these tests?
+    //    "single dots in id '/test/.' pass schema and validation" in {
+    //      testInvalid("/test/.")
+    //    }
+    //
+    //    "single dots in id '/./not.point.less' pass schema and validation" in {
+    //      testInvalid("/./not.point.less")
+    //    }
+
+    //    // non-absolute paths (could be allowed in some contexts)
```

Comment:
Shouldn't this and the next test still be valid for a relative `PathId`?

Reply:
I've converted the tests to use raml.App, as we can't construct AppDefinitions with relative Paths.

I've also added some tests, please have another pass if they're correct. I think they should be, but... eh.